scripts/reptiles.js

```diff
@@ -0,0 +1,30 @@
+let mainDiv = document.querySelector('main');
+
+let containerDiv = document.createElement('div');
+containerDiv.classList.add('container');
+mainDiv.appendChild(containerDiv);
+
+let imageElement = document.createElement('img');
```

Comment:
Any element where you're fetching/creating something using document.querySelector/getElementById etc, then it should be created with Const as you will never modify this value.

The use of let is used for when you're changing something, such as a variable turning true/false, or any other value that should change when its used.

Reply:
Also this is a lot of code, maybe consider making this a function?